common/aws/common.go

```diff
@@ -25,22 +25,38 @@ import (
 )
 
 const (
-	// ResourceType is the Boskos Type for AWS accounts
-	ResourceType = "aws-account"
-
 	// UserDataAccessIDKey is the key in UserData containing the AWS Access key ID
 	UserDataAccessIDKey = "access-key-id"
 
 	// UserDataSecretAccessKey is the key in UserData containing the AWS Secret access key
 	UserDataSecretAccessKey = "secret-access-key"
 )
 
+func GetSupportedAWSBoskosResourceTypes() []string {
+	// Supported Resource Types for Boskos AWS accounts
+	return []string{
+		"aws-account",
+		"image-builder-aws-account",
+	}
+}
+
+func checkResourceType(resource string, supportedResourceTypes []string) bool {
+	for _, supportedResource := range supportedResourceTypes {
```

Comment:
maybe we could require the resource type to end with `aws-account` as an invariant instead?

Reply:
ooh, I like that idea